examples/sdbus_async/rfcomm-serial-connect.py

```diff
@@ -0,0 +1,114 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+# Copyright (C) 2022  Shin'ichiro Kawasaki <kawasaki@juno.dti.ne.jp>
+from __future__ import annotations
+
```

Comment:
Would you mind adding a description of what the code is doing.

From reading it looks like it registers a RFCOMM protocol which provides an TCP-like connection between bluetooth devices? (sorry my Bluetooth knowledge is limited)

Reply:
I will try to add some descriptions. My bluetooth knowledge is still limited and needs more learning too :)

My understanding is that blueoothd handles all socket connection boiler plates. Using D-Bus bluez Profile and Profile Manager interface, applications can just request connection to the bluetooth RFCOMM service of a profile device, and receive a file descriptor which ready for read and write.

I agree that Profile and Profile Manager interface usage is complicated. I myself struggled to get basic understanding to write this example. I could not find good description about the interfaces on the net. What I found was another one who struggled to understand it, and wrote [another example in C++](https://github.com/5cript/bluez-rfcomm-example).  That's why I think this example is valuable for other python-sdbus-bluez users.

Reply:
I've added descriptions in the example script and pushed to the branch. Comments on the descriptions for further improvement will be welcome.